src/Worker/BuildWorker.php

```diff
@@ -276,6 +277,11 @@ protected function verifyJob(Job $job)
 
             return false;
         }
+        if (self::JOB_ID_STOP === $jobData['build_id']) {
+            $this->stopWorker();
+
+            return false;
+        }
 
         return true;
     }
```

Comment:
Also would be great if you add short description of the feature to documentation. Thanks!

Reply:
in docs/en/workers/worker.md ? What chapter title ?

Reply:
Yes. Could you create new chapter. Something like "Safe stopping worker"?